STUDIO/org.openl.rules.workspace/pom.xml

```diff
@@ -26,6 +26,17 @@
             <artifactId>xmlunit-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <!-- Annotation classes are used in generated from Datatype Table classes.
+            To compile them without adding extra dependency in client's project "compile" scope is used -->
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+            <version>2.3.5</version>
+        </dependency>
```

Comment:
Move under to         <!-- OpenL dependencies -->    section
Remove copy-pasted dependency. Fix version